troubleshooting/find-a-failed-submission.md

```diff
@@ -80,22 +80,60 @@ The payload is encrypted. Unfortunately you will need to decrypt it before you c
 submission.update(payload: EncryptionService.new.encrypt(submission.decrypted_payload))
 ~~~~~~~~
 
-There are 2 rake tasks that can be run in order to replay a Delayed Job. These will need to be run from outside the Rails console, but still on the Submitter container, in the normal way. This first replays a submission without any attachments. So for a submission with the ID '12345':
+There are 3 rake tasks that can be run in order to replay Delayed Jobs. These will need to be run from outside the Rails console, but still on the Submitter container, in the normal way. One replays submissions without any attachments. So for a submission with the ID '12345':
 
 ~~~~~~~~
-bundle exec rake replay_submission:process[12345]
+bundle exec rake replay_submissions:process[12345]
 ~~~~~~~~
 
-The second is for replaying submissions which have attachments. The filestore only validates JWT's that were created 60 seconds or less ago. So to replay any submissions with attachments after that we need to override the leeway using the rake task:
+You can also replay several submissions by separating them with spaces
 
 ~~~~~~~~
-bundle exec rake replay_submission:with_attachments[12345,600]
+bundle exec rake replay_submissions:process[12345 67890]
 ~~~~~~~~
 
+Another is for replaying submissions which have attachments. The filestore only validates JWT's that were created 60 seconds or less ago. So to replay any submissions with attachments after that we need to override the leeway using the rake task:
+
+~~~~~~~~
+bundle exec rake replay_submissions:with_attachments[12345,600]
```

Comment:
just to clarify, is this overriding the 60 seconds leeway with 600 seconds?

Reply:
Yes it is. I shall change that number to something else so it's clear